Sources/MQ/Errors/TheErrorGroup.swift

```diff
@@ -71,4 +71,25 @@ extension TheErrorGroup {
 	) -> Identifier? {
 		self.identifiers.first(where: matches)
 	}
+
+	internal func isSubset(
+		of other: Self
+	) -> Bool {
+		for identifier in self.identifiers where other.identifiers.contains(identifier) {
```

Comment:
Where is false ?

Reply:
removed by formatter 🙈 fixed by using guard